tests/test_security_files.py

```diff
@@ -0,0 +1,7 @@
+def test_insecure_post_fails(flask_test_client):
```

Comment:
I am not very sharp - do you think you could explain this test a little bit more? Thank you.